src/Apps/MyCollection/Routes/EditArtwork/__tests__/MyCollectionArtworkForm.jest.tsx

```diff
@@ -400,6 +411,15 @@ describe("Edit artwork", () => {
         })
       )
 
+      expect(trackingSpy).toHaveBeenCalledWith({
```

Comment:
**suggestion:** although it's okay here to compare object, comparing snapshots as well would be also fine here